src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java

```diff
@@ -73,8 +73,8 @@ public class GuiManager implements Manager, GameScreenAPI {
     private enum LoadStatus {
         WAITING(gm -> gm.main.hero.address != 0 && !gm.connecting.isVisible()),
         AFTER_LOGIN(gm -> {
-            API.directKeyClick(gm.main.config.LOOT.AMMO_KEY);
-            API.directKeyClick(gm.main.config.LOOT.AMMO_KEY);
+            API.rawKeyboardClick(gm.main.config.LOOT.AMMO_KEY, false);
+            API.rawKeyboardClick(gm.main.config.LOOT.AMMO_KEY, false);
```

Comment:
use `API.keyboardClick(Character, boolean)` (it doesn't exist, you'll have to add it, but `raw` keyboard clicks should be an internal thing, not needed by outside callers)